catalog.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+<body>
+
+</body>
+<header>
+
+  <nav>
+    <a href="">
+      <img src="https://via.placeholder.com/150x100" alt="">
+    </a>
+    <ul>
+      <li><a href="">информация</a></li>
+      <li><a href="">фото и видео</a></li>
+      <li><a href="">карта штата</a></li>
+      <li><a href="">гостиницы</a></li>
+    </ul>
+  </nav>
+</header>
+<main>
+  <section>
+    <h2></h2>
+    Инфраструктура
+    Бассейн
+    Парковка
+    Wi-Fi
+    Тип Жилья
+    Гостиница
+    Мотель
+    Апартаменты
+    Стоимость в сутки(Р)
+    от 0 до 3000
+    Показать
+  </section>
+  <section>
+    <h2></h2>
+    Найдено: 3
+    Сортировка:
+    По цене
+    По типу
+    По рейтингу
+    <ul>
+      <li>
+        <h3>Amara resort & spa</h3>
```

Comment:
саму карточку товара можно обернуть в тег article, который будет внутри лишки

Reply:
заголовок должен быть ссылкой, вот так:
```suggestion
        <h3><a href="#!">Amara resort & spa</a></h3>
```